third_party/silabs/SiWx917_sdk.gni

```diff
@@ -330,6 +397,22 @@ template("siwx917_sdk") {
       "${sdk_support_root}/matter/mbedtls/tinycrypt/src/x509write_csr.c",
     ]
 
+
+      # nvm3 ans startup
+      if ( silabs_board == "BRD4325B" ) {
+      sources += [
+        "${wifi_sdk_root}/components/siwx917_soc/drivers/middleware/nvm3/src/sl_si91x_dual_flash_intf.c",
+        "${examples_plat_dir}/device/src/startup_RS1xxxx.c",
+        ]
+      }
+
+      if (silabs_board == "BRD4325C" ) {
```

Comment:
Does the default file works with BRD4325G?

Reply:
Yes BRD4325C file works with BRD4325G